.github/workflows/ci.yml

```diff
@@ -319,30 +319,30 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: actions/setup-python@v2
         with:
-          activate-environment: modin
-          environment-file: environment-dev.yml
-          python-version: 3.7
-          channel-priority: strict
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-      - name: Conda environment
-        shell: bash -l {0}
-        run: |
-          conda info
-          conda list
-
+          python-version: "3.7.x"
+          architecture: "x64"
+      - name: ASV installation
+        run: pip install git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
```

Comment:
can we use a tagged something instead of raw commit?..

Reply:
There is no such possibility - the previous release was about a year ago, it is not known when the next one will be.
https://github.com/airspeed-velocity/asv/issues/967

Reply:
Please add a comment anywhere then describing the reasons (and add a `FIXME` to use a tag when a release is done).

Reply:
Added